Formula/git.rb

```diff
@@ -7,6 +7,7 @@ class Git < Formula
   head "https://github.com/git/git.git", :shallow => false
 
   bottle do
+    revision 1
```

Comment:
This should be outside the `bottle` block. Usual position is between the main `sha256` (line 6) and `head` (line 7).

Reply:
thanks. I'll change that in v2.